DOCKER/Dockerfile.develop

```diff
@@ -0,0 +1,35 @@
+FROM alpine:3.5
+
+ENV DATA_ROOT /tendermint
+ENV TMROOT $DATA_ROOT
+
+RUN addgroup tmuser && \
+    adduser -S -G tmuser tmuser
+
+RUN mkdir -p $DATA_ROOT && \
+    chown -R tmuser:tmuser $DATA_ROOT
+
+RUN apk add --no-cache bash curl jq
+
+ENV GOPATH /go
+ENV PATH "$PATH:/go/bin"
+RUN mkdir -p /go/src/github.com/tendermint/tendermint && \
```

Comment:
what do you think about doing something like https://github.com/tendermint/tendermint/blob/master/test/docker/Dockerfile#L9 here also to take advantage of caching ?